web/src/app/pages/create-survey/name-survey/name-survey.component.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Component, Input, OnChanges} from '@angular/core';
+import {FormGroup, FormBuilder} from '@angular/forms';
+
+@Component({
+  selector: 'name-survey',
```

Comment:
Ditto on the naming of the component. This is a little difficult to parse, and feels restrictive. Maybe something like survey-details-form instead?

Reply:
Sg, renamed to `survey-details` to keep it short.